arm-relay/2017-04-01/swagger/relay.json

```diff
@@ -1575,9 +1581,6 @@
           "description": "Resource tags"
         }
       },
-      "required": [
```

Comment:
Have added a new Namespace update parameters, Resource for namespace patch and added back location required to TrackedResource. Please review and let me know your views.